server/migrations/20181113025401-update-articleid-column.js

```diff
@@ -0,0 +1,15 @@
+export default {
+  up: queryInterface => queryInterface.sequelize.query('alter table "Comments" drop constraint "Comments_articleId_fkey"')
+    .then(() => queryInterface.sequelize.query(
```

Comment:
whyyyyy?

Reply:
Whats the advantage of writing raw queries here?

Reply:
The Sequelize `updateColumn` method did not work. For some reason, it adds the updated foreign key to the column without removing the old foreign key attribute.
Raw queries solved this problem.